{
  "id" : "1f98f85c-ea93-4ead-b6ed-09bb1be6c8b2",
  "name" : "api_2.0_sql_statements",
  "request" : {
    "url" : "/api/2.0/sql/statements/",
    "method" : "POST",
    "bodyPatterns" : [ {
      "equalToJson" : "{\"statement\":\"INSER INTO main.jdbc_test_schema.query_syntax_error_test_table (id, col1, col2) VALUES (1, 'value1', 'value2')\",\"warehouse_id\":\"dd43ee29fedd958d\",\"session_id\":\"01efe861-09a5-1816-8b4a-74a1a324f95a\",\"disposition\":\"EXTERNAL_LINKS\",\"format\":\"ARROW_STREAM\",\"on_wait_timeout\":\"CONTINUE\",\"wait_timeout\":\"10s\",\"parameters\":[],\"result_compression\":\"LZ4_FRAME\"}",
      "ignoreArrayOrder" : true,
      "ignoreExtraElements" : true
    } ]
  },
  "response" : {
    "status" : 200,
    "body" : "{\"statement_id\":\"01efe861-0a6d-161e-9138-2e488cbacea0\",\"status\":{\"state\":\"FAILED\",\"error\":{\"error_code\":\"BAD_REQUEST\",\"message\":\"\\n[PARSE_SYNTAX_ERROR] Syntax error at or near 'INSER'. SQLSTATE: 42601 (line 1, pos 0)\\n\\n== SQL ==\\nINSER INTO main.jdbc_test_schema.query_syntax_error_test_table (id, col1, col2) VALUES (1, 'value1', 'value2')\\n^^^\\n\"},\"sql_state\":\"42601\"}}",
    "headers" : {
      "x-request-id" : "8ad7d276-e49a-4662-b768-c8a646911b34",
      "date" : "Tue, 11 Feb 2025 10:14:55 GMT",
      "server" : "databricks",
      "x-databricks-popp-response-code-details" : "via_upstream",
      "x-databricks-shard-debug" : "oregon-staging",
      "vary" : "Accept-Encoding",
      "x-databricks-org-id" : "6051921418418893",
      "strict-transport-security" : "max-age=31536000; includeSubDomains; preload",
      "x-content-type-options" : "nosniff",
      "x-databricks-popp-routing-reason" : "deployment-name",
      "content-type" : "application/json",
      "x-databricks-apiproxy-response-code-details" : "via_upstream"
    }
  },
  "uuid" : "1f98f85c-ea93-4ead-b6ed-09bb1be6c8b2",
  "insertionIndex" : 24
}